### PR TITLE
CNV-13793: Release notes change, supported guest OS

### PR DESCRIPTION
**Pull request by sabrinajess**
Re: [CNV-13793](https://issues.redhat.com/browse/CNV-13793) (first commit)
Re: [CNV-11664](https://issues.redhat.com/browse/CNV-11664) (second commit)
For 4.9 release notes

Preview: https://deploy-preview-36347--osdocs.netlify.app/openshift-enterprise/latest/virt/virt-4-9-release-notes.html

### CODE REVIEW

**Comment by thenaggster** [comment]
lgtm
Confirmed with Martin Tessun.

**Review by thenaggster — APPROVED** [review]
lgtm

File: virt/virt-4-9-release-notes.adoc [hunk 1]
```diff
@@ -23,6 +23,8 @@ include::modules/virt-supported-cluster-version.adoc[leveloffset=+2]
 {VirtProductName} guests can use the following operating systems:
 
 * Red Hat Enterprise Linux 6, 7, and 8.
+* Red Hat Enterprise Linux 9 Alpha (TP) and Red Hat Enterprise Linux 9 Beta (TP).
```
Comment (rnetser) [inline virt/virt-4-9-release-notes.adoc:26]:
```suggestion
* Red Hat Enterprise Linux 9 Alpha (TP).
```
In CNV 4.9.0 we provide a template for Rhel9 Alpha (which is the OS that was tested).

File: virt/virt-4-9-release-notes.adoc [hunk 2]
```diff
@@ -23,6 +23,8 @@ include::modules/virt-supported-cluster-version.adoc[leveloffset=+2]
 {VirtProductName} guests can use the following operating systems:
 
 * Red Hat Enterprise Linux 6, 7, and 8.
+* Red Hat Enterprise Linux 9 Alpha (TP).
+* Fedora 31 and 32.
```
Comment (ILpinto) [inline virt/virt-4-9-release-notes.adoc:27]: We support Fedora 34 
annotations:
defaults.template.kubevirt.io/disk: rootdisk
description: 'Template for Fedora 32 VM or newer. A PVC with the Fedora disk image
must be available. Recommended disk image: https://download.fedoraproject.org/pub/fedora/linux/releases/32/Cloud/x86_64/images/Fedora-Cloud-Base-32-1.6.x86_64.qcow2'
iconClass: icon-fedora
name.os.template.kubevirt.io/fedora32: Fedora 32 or higher
name.os.template.kubevirt.io/fedora33: Fedora 32 or higher
name.os.template.kubevirt.io/fedora34: Fedora 32 or higher
name.os.template.kubevirt.io/silverblue32: Fedora 32 or higher
name.os.template.kubevirt.io/silverblue33: Fedora 32 or higher
name.os.template.kubevirt.io/silverblue34: Fedora 32 or higher
openshift.io/display-name: Fedora 32+ VM
openshift.io/documentation-url: https://github.com/kubevirt/common-templates
openshift.io/provider-display-name: KubeVirt
openshift.io/support-url: https://github.com/kubevirt/common-templates/issues
operator-sdk/primary-resource: openshift-cnv/ssp-kubevirt-hyperconverged
operator-sdk/primary-resource-type: SSP.ssp.kubevirt.io
tags: hidden,kubevirt,virtualmachine,fedora

Comment (ILpinto) [reply virt/virt-4-9-release-notes.adoc:27]: After checking with PM. we should not mention Fedora as supported OS. please remove it.

**Review by sjhala-ccs — COMMENTED** [review]
Looks good, one suggestion for you to consider.

**Comment by sabrinajess** [comment]
@dominikholler PRs need to go through peer review after QE and prior to merge.

**Pull request merged**